free-2.md

```diff
@@ -58,12 +135,34 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+Since these criteria are most easily described in natural language, it would
+be a significant challenge translating it into code.
+So a lot of time would need to be dedicated to syntax and naming.
+
```

Comment:
I would also say that creating an effective natural language would be difficult so the language aspects will require time.